data/cybox/nozomi/nozomi_alerts_22022024.json

```diff
@@ -0,0 +1,563 @@
+                        }
+                    },
+                    "name": "Hardware Additions",
+                    "kill_chain_phases": [
+                        {
+                            "kill_chain_name": "mitre-attack",
+                            "phase_name": "Initial Access"
+                        }
+                    ]
+                }
+            },
+            "first_observed": "2024-01-09T09:19:33.000Z",
+            "last_observed": "2024-01-09T09:19:33.000Z",
+            "number_observed": 1
+        },
+        {
+            "id": "observed-data--cb5f268e-73b8-4a39-82a2-67d87d8ce62b",
+            "type": "observed-data",
+            "created_by_ref": "identity--f431f809-377b-45e0-aa1c-6a4751cae5ff",
+            "created": "2024-02-22T06:51:20.244Z",
+            "modified": "2024-02-22T06:51:20.244Z",
+            "objects": {
+                "0": {
+                    "type": "x-ibm-finding",
+                    "alert_id": "57ee3f7d-80eb-54db-aabf-8199d3b872d8",
+                    "x_is_acknowledged": false,
+                    "severity": 70,
+                    "time_observed": "2024-01-09T09:17:29.000Z",
+                    "dst_ip_ref": "1",
+                    "src_ip_ref": "3",
+                    "x_alert_status": "open",
+                    "name": "Malformed traffic",
+                    "ttp_tagging_refs": [
+                        "8"
+                    ],
+                    "description": "The UDP header in the packet contains a wrong data length. 1404 bytes are advertised, but 1380 bytes are found in the payload.",
+                    "x_is_incident_alert": false,
+                    "x_is_cybersecurity_alert": true,
+                    "finding_type": "alert",
+                    "start": "2024-01-09T09:17:29.000Z",
+                    "x_sensor_host": "Demo Sensor standard 4959963c",
+                    "x_sensor_interface": "em1",
+                    "x_alert_type_id": "SIGN:MALFORMED-TRAFFIC",
+                    "x_rule_id": "udp.wrong-length"
+                },
+                "1": {
+                    "type": "ipv4-addr",
+                    "value": "7.7.7.7",
+                    "resolves_to_refs": [
+                        "4"
+                    ],
+                    "x_nozomi_info_ref": "6"
+                },
+                "2": {
+                    "type": "network-traffic",
+                    "dst_ref": "1",
+                    "src_ref": "3",
+                    "protocols": [
+                        "unknown"
+                    ]
+                },
+                "3": {
+                    "type": "ipv4-addr",
+                    "value": "8.8.8.8",
+                    "resolves_to_refs": [
+                        "5"
+                    ],
+                    "x_nozomi_info_ref": "7"
+                },
+                "4": {
+                    "type": "mac-addr",
+                    "value": "ff:ff:xx:ff:ff:ff",
```

Comment:
Please use a valid but clearly false MAC address, such as "01:01:01:01:01:01"

Reply:
Updated as per the suggestion.